gulpfile.js

```diff
@@ -1,4 +1,5 @@
 var gulp = require('gulp');
+var pump = require('pump');
```

Comment:
This lib takes an array of streams and a "finished" function, pipes the streams together, and calls finishes when the pipeline has finished, or any of the streams error.
```js
pump([streamA, streamB], function (err) {
  // done!
})
```